Userland/Libraries/LibCore/Group.cpp

```diff
@@ -48,11 +49,11 @@ ErrorOr<void> Group::add_group(Group& group)
     if (!file)
         return Error::from_errno(errno);
 
+    ScopeGuard file_guard = [file] { fclose(file); };
```

Comment:
We usually use uniform initialisation syntax for `ScopeGuard`s

Reply:
@alimpfard Sorry for being a bit dense, but like this?
```cpp
ScopeGuard file_guard = [&] { fclose(file); };
```

Reply:
Updated as above. This seems to be the most used pattern throughout the code base for ScopeGuards

Reply:
I found some code from alimpfard in the code base using uniform initialization syntax
```cpp
ScopeGuard file_guard { [&] {
    fclose(file);
} };
```